repository/w.json

```diff
@@ -109,6 +109,17 @@
 				}
 			]
 		},
+		{
+			"name": "WebAgent Syntax",
```

Comment:
You can just call it "WebAgent": languages that implement support for a specific language are simply called after that language.

Reply:
fixed